tests/test-buttercup.el

```diff
@@ -501,17 +501,40 @@
 
 (describe "The Spy "
   (let (test-function)
-    (before-each
+    (before-all
```

Comment:
Why this change? If one of the tests breaks the binding, all tests would fail now – that's not a particularly useful co-dependence. And the setup is not really expensive.